components/dashboard/wallet/Balance.js

```diff
@@ -25,26 +25,29 @@ const CurrencyBalances = (props) => {
       style={{ width: '100%', height: '100%' }}
     >
       {/* title row */}
-      <div
+      {/* <div
         className="d-flex flex-row justify-content-between align-items-start"
         style={{ height: '15%', width: '100%', marginBottom: '15px' }}
       >
         <p style={{ fontSize: '20px', fontWeight: '700', color: 'white' }}>
           Currency Balances
         </p>
         <button type="button">Add Token</button>
```

Comment:
If these code are no longer needed in the future then i suggest cleanup is needed or WDYT